src/sb_pointcloud_processing/launch/line_extractor_rviz_degree_3.launch

```diff
@@ -0,0 +1,23 @@
+<launch>
+    <node name="line_extractor_node" pkg="sb_pointcloud_processing" type="line_extractor_node" output="screen">
+        <!-- degree of polynomial for line of best fit -->
+        <param name="degree_polynomial" value="5" type="int" />
+        <!-- regularization constant for line of best fit -->
+        <param name="lambda" value="0" type="double" />
+        <!-- density parameters for DBSCAN -->
+        <param name="min_neighbours" value="10" type="int" />
+        <param name="radius" value="5" type="double" />
+    </node>
+    <node name="test_pcl_generator_node" pkg="sb_pointcloud_processing" type="test_pcl_generator_node" output="screen">
+        <rosparam param="first_line">[30, 0, -0.01, 0.0002]</rosparam>
+        <rosparam param="second_line">[0, 0, -0.01, 0.0002]</rosparam>
+        <param name="x_min" value="-40" type="double" />
+        <param name="x_max" value="60" type="double" />
```

Comment:
I might suggest some more representative values here (perhaps on the order of 5-10m?)

Reply:
done